JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -121,15 +119,9 @@ private fun BottomBarAction(
     @DrawableRes id: Int,
     onClick: () -> Unit
 ) {
-    Ripple(
-        bounded = false,
-        radius = 24.dp
-    ) {
-        Clickable(onClick = onClick) {
-            Container(modifier = LayoutPadding(12.dp) + LayoutSize(24.dp, 24.dp)) {
-                DrawVector(vectorResource(id))
-            }
-        }
+    IconButton(onClick = onClick, modifier = Modifier.padding(12.dp).preferredSize(24.dp, 24.dp)) {
+        // use Image instead of Icon to retain coloring from vector graphic
```

Comment:
Shouldn't the coloring be `contentColor()`?

Reply:
This was to preserve existing coloring, since contentColor defaults to black without additional configuration via ambient. Thoughts about keeping this or switching to Icon?